homeassistant/components/light/zwave.py

```diff
@@ -141,8 +148,41 @@ def supported_features(self):
         """Flag supported features."""
         return SUPPORT_ZWAVE_DIMMER
 
+    def _set_duration(self, **kwargs):
+        """Set the transition time for the brightness value.
+
+        Zwave Dimming Duration values:
+        0x00      = instant
+        0x01-0x7F = 1 second to 127 seconds
+        0x80-0xFE = 1 minute to 127 minutes
+        0xFF      = factory default
+        """
+        if self.values.dimming_duration is None:
+            if ATTR_TRANSITION in kwargs:
+                _LOGGER.debug("Dimming not supported by %s.", self.entity_id)
+            return
+
+        if ATTR_TRANSITION not in kwargs:
+            self.values.dimming_duration.data = 0xFF
```

Comment:
Here it is. You can see where the `dimming_duration` is part of the `SwitchMultilevelCmd_Set` message to the network.
```
2017-04-01 15:41:20.398 Info, Node003, Value::Set - COMMAND_CLASS_SWITCH_MULTILEVEL - Dimming Duration - 5 - 1 - \FF
2017-04-01 15:41:20.398 Detail, Node003, Refreshed Value: old value=255, new value=255, type=byte
2017-04-01 15:41:20.398 Detail, Node003, Changes to this value are not verified
2017-04-01 15:41:20.398 Detail, Node003, Notification: ValueChanged
2017-04-01 15:41:20.398 Info, Node003, Value::Set - COMMAND_CLASS_SWITCH_MULTILEVEL - Level - 0 - 1 - 
2017-04-01 15:41:20.398 Info, Node003, SwitchMultilevel::Set - Setting to level 0
2017-04-01 15:41:20.398 Info, Node003,   Duration: Default
2017-04-01 15:41:20.399 Detail, Node003, Setting Encryption Flag on Message For Command Class COMMAND_CLASS_SWITCH_MULTILEVEL
2017-04-01 15:41:20.399 Detail, Node003, Queuing (Send) SwitchMultilevelCmd_Set (Node=3): 0x01, 0x0b, 0x00, 0x13, 0x03, 0x04, 0x26, 0x01, 0x00, 0xff, 0x25, 0x4d, 0x50
2017-04-01 15:41:20.399 Detail, Node003, Setting Encryption Flag on Message For Command Class COMMAND_CLASS_SWITCH_MULTILEVEL
2017-04-01 15:41:20.399 Detail, Node003, Queuing (Send) SwitchMultilevelCmd_Get (Node=3): 0x01, 0x09, 0x00, 0x13, 0x03, 0x02, 0x26, 0x02, 0x25, 0x4e, 0xab
2017-04-01 15:41:20.402 Detail, 
```